gradle-plugin/README.md

```diff
@@ -4,7 +4,14 @@ For usage of this plugin, see the documentation.
 
 # Build Instruction
 
+This Gradle project depends on the `dependencies` module through the Maven local repository.
+
 ```
+# At the root of the cloud-opensource-java project, first install the Maven projects to Maven local
+# repository.
+$ mvn install
```

Comment:
indent four spaces for code

Reply:
Done with 3-space indentation (as in boms/cloud-oss-bom/RELEASING.md; 4-space didn't work in GitHub's markdown viewer.)